docs/content/concepts/assets/software-defined-assets.mdx

```diff
@@ -306,23 +311,7 @@ def asset_with_version():
     return 100
 ```
 
-When an asset with a code version is materialized, the generated `AssetMaterialization` is tagged with the version. The UI will indicate when an asset has a different code version than the code version used for its most recent materialization.
-
-Multi-assets may assign different code versions for each of their outputs:
-
-```python file=/concepts/assets/code_versions.py startafter=start_multi_asset endbefore=end_multi_asset
-@multi_asset(
-    outs={
-        "a": AssetOut(code_version="1"),
-        "b": AssetOut(code_version="2"),
-    }
-)
-def multi_asset_with_versions():
-    yield Output(100, "a")
-    yield Output(200, "b")
-```
-
-Just as with regular assets, these versions are attached to the `AssetMaterialization` objects for each of the constituent assets and represented in the UI.
+When an asset with a code version is materialized, it is tagged with the version. The UI will indicate when an asset has a different code version than the code version used for its most recent materialization.
```

Comment:
removed the `AssetMaterialization` here since this is the first mention of it and i could see that being confusing. Additionally, i don't think it's that important that the user knows we tag the `AssetMaterialization` event (unless their like a power user maybe) but open to push-back on that conclusion